BeVolt/BSP/Simulator/DataGeneration/Data/CAN.csv

```diff
@@ -0,0 +1 @@
+0x103,0x20,0x20,0x20,0x20,0x20,0x20,0x20,0x20

```

Comment:
Can you remove this file? Ideally, this file will only be stored locally and never committed

Reply:
I agree.